doc/src/guides/solving/solve-matrix-equation.md

```diff
@@ -0,0 +1,341 @@
+    ₃⋅A₂₁⋅A₃₂ - A₀₀⋅A₁₃⋅A₂₂⋅A₃₁ - A₀₁⋅A₁₀⋅A₂₂⋅A₃₃ + A₀₁⋅A₁₀⋅A₂₃⋅A₃₂ + A₀₁⋅A₁₂⋅A₂₀⋅
+    A₃₃ - A₀₁⋅A₁₂⋅A₂₃⋅A₃₀ - A₀₁⋅A₁₃⋅A₂₀⋅A₃₂ + A₀₁⋅A₁₃⋅A₂₂⋅A₃₀ + A₀₂⋅A₁₀⋅A₂₁⋅A₃₃ - 
+    A₀₂⋅A₁₀⋅A₂₃⋅A₃₁ - A₀₂⋅A₁₁⋅A₂₀⋅A₃₃ + A₀₂⋅A₁₁⋅A₂₃⋅A₃₀ + A₀₂⋅A₁₃⋅A₂₀⋅A₃₁ - A₀₂⋅A₁
+    ₃⋅A₂₁⋅A₃₀ - A₀₃⋅A₁₀⋅A₂₁⋅A₃₂ + A₀₃⋅A₁₀⋅A₂₂⋅A₃₁ + A₀₃⋅A₁₁⋅A₂₀⋅A₃₂ - A₀₃⋅A₁₁⋅A₂₂⋅
+    A₃₀ - A₀₃⋅A₁₂⋅A₂₀⋅A₃₁ + A₀₃⋅A₁₂⋅A₂₁⋅A₃₀
+```
+
+and solving a matrix equation of it takes about a minute, whereas the analogous
+3x3 matrix takes less than one second. The more unrelated, symbolic entries in a
+matrix, the more likely it is to be slow to manipulate. This example, finding a
+general solution to a matrix where all elements are independent symbols, is the
+extreme case and thus the slowest for a matrix of its size.
+
+One way to speed up matrix operations is by using the {class}`~.DomainMatrix`
+class, which can be faster to operate on because it limits the domain of matrix
+elements.
+
+## Use the Solution Result
+
+### Use the Solution as a Vector
+
+You can use the solution result as a vector. For example, to prove that the
+solution $x$ is correct, you can multiply it the matrix $A$ and verify that it
+produces the constants vector $b$:
+
+```py
+>>> from sympy import symbols, simplify
+>>> from sympy.matrices import Matrix
+>>> c, d, e = symbols("c, d, e")
+>>> A = Matrix([[c,d], [1, -e]])
+>>> b = Matrix([2, 0])
+>>> solution = A.solve(b)
+>>> solution
+    ⎡  2⋅e  ⎤
+    ⎢───────⎥
+    ⎢c⋅e + d⎥
+    ⎢       ⎥
+    ⎢   2   ⎥
+    ⎢───────⎥
+    ⎣c⋅e + d⎦
+
+>>> (A * solution) - b # Not immediately obvious whether this result is a zeroes vector
+    ⎡ 2⋅c⋅e      2⋅d      ⎤
+    ⎢─────── + ─────── - 2⎥
+    ⎢c⋅e + d   c⋅e + d    ⎥
+    ⎢                     ⎥
+    ⎣          0          ⎦
+>>> simplify((A * solution) - b) # simplify reveals that this result is a zeroes vector
+    ⎡0⎤
+    ⎢ ⎥
+    ⎣0⎦
+```
+
+Note that we had to use {func}`~sympy.simplify.simplify.simplify` to make SymPy
+simplify the expression in a matrix element to make it immediately obvious that
+the solution is correct.
+
+### Extract Elements From the Solution
+
+Because you can iterate through the elements in a column vector, you can extract
+its elements using standard Python techniques. For example, you can create a
+list of the elements using list comprehension
+
+```py
+>>> [element for element in solution]
+    ⎡  2⋅e       2   ⎤
+    ⎢───────, ───────⎥
+    ⎣c⋅e + d  c⋅e + d⎦
+```
+
+or you can extract individual elements by subscripting ("slicing")
+
+```py
+>>> solution[0]
+      2⋅e  
+    ───────
+    c⋅e + d
+```
+
+## Equations With No Solution
+
+If the determinant of a matrix is zero, matrix equations with it have no
+solution:
+
+```py
+>>> from sympy import symbols
+>>> from sympy.matrices import Matrix
+>>> c, d, e = symbols("c, d, e")
+>>> A = Matrix([[c*e**2, d*e], [c*e, d]])
+>>> A
+    ⎡   2     ⎤
+    ⎢c⋅e   d⋅e⎥
+    ⎢         ⎥
+    ⎣c⋅e    d ⎦
+>>> b = Matrix([2, 0])
+>>> A.LUsolve(b)
```

Comment:
Have you introduced this above? Probably should explain the difference in `.solve()` somewhere before.

Reply:
Per your other comment about `LUsolve`, I now introduce `LUsolve` earlier.

> Probably should explain the difference in `.solve()` somewhere before.

I added a section [Methods for Solving Matrix Equations](https://output.circle-artifacts.com/output/job/d9bcb4f2-c266-442f-a61e-f1bec15ed69e/artifacts/0/doc/_build/html/guides/solving/solve-matrix-equation.html#methods-for-solving-matrix-equations), noting that Gauss-Jordan elimination is the default method, and another method is LU decomposition (via `solve` with `method='LU'`, or `LUsolve`).

I think my earlier query to you, which method is best for a given kind of matrix (hermitian, symmetric, or triangular), will address this question?